openbts/components.py

```diff
@@ -41,6 +42,37 @@ def monitor(self):
     }
     return self._send_and_receive(message)
 
+  def tmsis(self, fields=['IMSI', 'ACCESSED'], access_period=0, auth=2):
+    """Gets all active subscribers from the TMSI table.
+
+    Args: access_period: fetches all entries with ACCESS < acess_period
+                         (default=0 filter off)
+          auth: fetches all entries with AUTH = auth
+                         (default=2)
+
+    Returns: a list of objects defined by the list of fields.
```

Comment:
I see your point with this one, better to keep it generic and take what we need on the client side